apps/api/src/server/applications/examination-timetable-items/__tests__/examination-timetable-items.test.js

```diff
@@ -255,16 +421,30 @@ describe('Test examination timetable items API', () => {
 
 	test('publish examination timetable item publishes examination items for the case', async () => {
 		databaseConnector.case.findUnique.mockResolvedValue({ id: 123 });
+
+		databaseConnector.examinationTimetable.update.mockResolvedValueOnce({ id: 123 });
+
+		databaseConnector.examinationTimetableItem.findMany.mockResolvedValueOnce(
+			publishExaminationTimetableItemsData
+		);
+
 		const resp = await request
 			.patch('/applications/examination-timetable-items/publish/123')
 			.send({});
+
 		expect(databaseConnector.examinationTimetable.update).toHaveBeenCalledWith({
 			where: {
 				caseId: 123
 			},
 			data: { published: true, publishedAt: expect.any(Date), updatedAt: expect.any(Date) }
 		});
 
+		expect(eventClient.sendEvents).toHaveBeenLastCalledWith(
+			NSIP_EXAM_TIMETABLE,
+			expectedPublishExaminationTimetableItemsPayload,
+			'Publish'
```

Comment:
would have expected to use EventType.Publish rather than string, using
import { EventType } from '@pins/event-client';

Reply:
Done